airflow/providers/amazon/aws/executors/ecs/ecs_executor.py

```diff
@@ -42,10 +42,14 @@
     EcsQueuedTask,
     EcsTaskCollection,
 )
-from airflow.providers.amazon.aws.executors.utils.exponential_backoff_retry import exponential_backoff_retry
+from airflow.providers.amazon.aws.executors.utils.exponential_backoff_retry import (
+    calculate_next_attempt_delay,
+    exponential_backoff_retry,
+)
 from airflow.providers.amazon.aws.hooks.ecs import EcsHook
 from airflow.utils import timezone
 from airflow.utils.state import State
+from airflow.utils.timezone import utcnow
```

Comment:
Some small nit, we've already import entire airflow.utils.timezone module, so potentially instead of `utcnow` we could use `timzeone.utcnow`

Reply:
Yeah, you're right. I'll make the change.

Reply:
Yeah, you're right. I'll make the change.

Reply:
Change applied in https://github.com/apache/airflow/pull/37109/commits/18a351b46be85f48f1c5b9fbe60b8c2fd64dc3d6